cassandra-operator/test/e2e/cluster_status.go

```diff
@@ -188,7 +188,7 @@ func GetMetric(metricGetterFunc func() (string, error), metricToFind string) (in
 		return 0, fmt.Errorf("failed to get metrics with %v", err)
 	}
 
-	if !(strings.Contains(metrics, metricToFind)){
+	if !(strings.Contains(metrics, metricToFind)) {
```

Comment:
```suggestion
	if !strings.Contains(metrics, metricToFind) {
```

I don't think the surrounding () are necessary in Go.

Reply:
Ok this was existing code but happy to change it!